src/threadinfo.h

```diff
@@ -13,7 +13,7 @@
 // For i386 and x86_64, SETJMP currently has bugs.  Don't turn this
 // on for them until they are debugged.
```

Comment:
In general, this commit remove setcontext/getcontext from RISC-V and you have a later commit, with some commit message like 'defining save_sp`.

The latter commit adds support for setjmp/longjmp, while this commit removes support for setcontext/getcontext.
Please take the line from the later commit for setjmp/longjmp, and add it to this commit.

Finally, then change the commit message to something like:
> riscv struct ThreadTLSInfo: setjmp, not setcontext
[ And note that this msg is exactly 50 chars ]